Makefile

```diff
@@ -22,12 +22,10 @@ help: ## display this help message
 	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
 
 clean: ## remove generated byte code, coverage reports, and build artifacts
-	find . -name '*.pyc' -exec rm -f {} +
-	find . -name '*.pyo' -exec rm -f {} +
-	find . -name '*~' -exec rm -f {} +
-	rm -fr build/
-	rm -fr dist/
-	rm -fr *.egg-info
+	find . -name '*.pyc' -delete
+	find . -name '*.pyo' -delete
+	find . -name '*~' -delete
```

Comment:
I'm just curious why you changed these find commands?

Reply:
`-delete` is noticeably faster. Otherwise, find has to run `rm -rf` for every file. I think that's why find has this common feature.

Reply:
That's why they used `+` instead of `;`: find will bunch up arguments to spawn fewer processes, but OK.